Description: Fix koch docs calling using git
Author: Ahmad Khalifa <ahmad@khalifa.ws>
Forwarded: not-needed

Remove git clone calls and expected cloned files

--- a/tools/kochdocs.nim
+++ b/tools/kochdocs.nim
@@ -94,7 +94,6 @@
   execFold(desc, cmd)
 
 const officialPackagesMarkdown = """
-pkgs/atlas/doc/atlas.md
 """.splitWhitespace()
 
 proc getMd2html(): seq[string] =
@@ -158,28 +157,9 @@
 """.splitWhitespace()
 
   officialPackagesList = """
-pkgs/asyncftpclient/src/asyncftpclient.nim
-pkgs/smtp/src/smtp.nim
-pkgs/punycode/src/punycode.nim
-pkgs/db_connector/src/db_connector/db_common.nim
-pkgs/db_connector/src/db_connector/db_mysql.nim
-pkgs/db_connector/src/db_connector/db_odbc.nim
-pkgs/db_connector/src/db_connector/db_postgres.nim
-pkgs/db_connector/src/db_connector/db_sqlite.nim
-pkgs/checksums/src/checksums/md5.nim
-pkgs/checksums/src/checksums/sha1.nim
-pkgs/checksums/src/checksums/sha2.nim
-pkgs/checksums/src/checksums/sha3.nim
-pkgs/checksums/src/checksums/bcrypt.nim
-pkgs/htmlparser/src/htmlparser.nim
 """.splitWhitespace()
 
   officialPackagesListWithoutIndex = """
-pkgs/db_connector/src/db_connector/mysql.nim
-pkgs/db_connector/src/db_connector/sqlite3.nim
-pkgs/db_connector/src/db_connector/postgres.nim
-pkgs/db_connector/src/db_connector/odbcsql.nim
-pkgs/db_connector/src/db_connector/private/dbutils.nim
 """.splitWhitespace()
 
 when (NimMajor, NimMinor) < (1, 1) or not declared(isRelativeTo):
@@ -348,7 +328,7 @@
 proc buildDocsDir*(args: string, dir: string) =
   let args = nimArgs & " " & args
   let docHackJsSource = buildJS()
-  gitClonePackages(@["asyncftpclient", "punycode", "smtp", "db_connector", "checksums", "atlas", "htmlparser"])
+  #gitClonePackages(@["asyncftpclient", "punycode", "smtp", "db_connector", "checksums", "atlas", "htmlparser"])
   createDir(dir)
   buildDocSamples(args, dir)
 
